### PR TITLE
Add PR Results page to Gubernator, showing a table of build results.

### PR DESCRIPTION
**Pull request by rmmh**

### CODE REVIEW

File: gubernator/static/style.css [hunk 1]
```diff
@@ -67,3 +68,18 @@ a.anchor, pre.error>a {
 a.anchor:target {
     color: blue;
 }
+table.pr {
+    margin: 20px;
```
Comment (ixdy) [inline gubernator/static/style.css]: minor ui nit: can we add a little bit of padding for the cell content?

Comment (rmmh) [inline gubernator/static/style.css]: Done.

File: gubernator/appengine_config.py [hunk 2]
```diff
@@ -15,10 +15,15 @@
...
 # Add any libraries installed in the "third_party" folder.
 vendor.add('third_party')
 
 # Use remote GCS calls for local development.
 if os.environ.get('SERVER_SOFTWARE','').startswith('Development'):
     os.environ['SERVER_SOFTWARE'] += ' remote_api'
+
+# The default deadline of 5 seconds is too aggressive of a target for GCS
+# directory listing operations.
+urlfetch.set_default_fetch_deadline(60)
```
Comment (spxtr) [inline gubernator/appengine_config.py]: :'(

Eventually we should move to some sort of database that can actually handle these queries reasonably quickly.

Comment (rmmh) [inline gubernator/appengine_config.py]: Agreed. Making users wait 5+ seconds for their page is not a good experience.

**Comment by spxtr** [comment]
It would be really awesome if this showed the git commits somehow, so that it's clear what's flakes. Not sure how's best to fit that in, though.

**Comment by rmmh** [comment]
I have some follow-up work to show specific revisions and the times that tests were run.

One re-implemented bit of testgrid at a time!

**Comment by spxtr** [comment]
Seems fine to me.

File: gubernator/appengine_config.py (hunk 2 above)
Comment (ixdy) [inline gubernator/appengine_config.py]: background task and/or memcache?

**Pull request merged**